app/App/Panel/index.js

```diff
@@ -69,7 +69,7 @@ class Panel extends React.Component {
 
     let gasPrice = this.store('main.networksMeta', type, id, 'gas.price.levels.fast')
     if (!gasPrice) gasPrice = this.store('main.networksMeta', type, id, 'gas.price.fees.maxFeePerGas')
-    if (gasPrice) gasPrice = Math.round(parseInt(gasPrice, 'hex') / 1e9)
+    if (gasPrice) gasPrice = parseInt(gasPrice, 'hex') / 1e9 < 1 ? '‹1' : Math.round(parseInt(gasPrice, 'hex') / 1e9)
```

Comment:
might be worth throwing this into its own function as the logic is getting a little complex